src/data/arena-tag.ts

```diff
@@ -832,13 +842,14 @@ class ToxicSpikesTag extends ArenaTrapTag {
           );
           return true;
         }
-      } else if (!pokemon.hasNonVolatileStatusEffect()) {
-        const toxic = this.layers > 1;
-        if (
-          pokemon.trySetStatus(!toxic ? StatusEffect.POISON : StatusEffect.TOXIC, true, null, 0, this.getMoveName())
-        ) {
-          return true;
-        }
+      } else {
+        return pokemon.trySetStatus(
+          !(this.layers > 1) ? StatusEffect.POISON : StatusEffect.TOXIC,
+          true,
+          null,
+          0,
+          this.getMoveName(),
+        );
```

Comment:
Sorry I only meant the if inside, I don't know if I'd want to get rid of the else if entirely, even though the result should be the same. As you prefer
```suggestion
      } else if (!pokemon.hasNonVolatileStatusEffect()) {
        const toxic = this.layers > 1;
        return pokemon.trySetStatus(toxic ? StatusEffect.TOXIC : StatusEffect.POISON, true, null, 0, this.getMoveName());
```

(If kept as is though I'd prefer if 
`!(this.layers > 1) ? StatusEffect.POISON : StatusEffect.TOXIC,`
got changed to 
`this.layers > 1 ? StatusEffect.TOXIC : StatusEffect.POISON,`

I think it's much clearer this way